executor/aggfuncs/func_max_min.go

```diff
@@ -627,7 +627,7 @@ func (e *maxMin4JSON) UpdatePartialResult(sctx sessionctx.Context, rowsInGroup [
 			continue
 		}
 		if p.isNull {
-			p.val = input
+			p.val = input.Copy()
```

Comment:
Does `value4JSON` need this `Copy` as well?

Reply:
Maybe not, because window function buffers all the rows belonging to the same partition during the calculation. After result is copied to the destination Chunk, the buffered Chunks are released. @lamxTyler Please confirm that.

Reply:
Yes, currently we do not reuse chunks. Maybe it is better to `copy` it to avoid future problems.